MIGRATIONS.md

```diff
@@ -22,6 +22,7 @@ User-facing changes are documented in the [changelog](CHANGELOG.md).
 
 ### Postgres Evolutions:
 - [037-add-publications.sql](conf/evolutions/037-add-publications.sql)
+- [038-more-voxel-types.sql](conf/evolutions/038-more-voxel-types.sql)
```

Comment:
@fm3 This landed in the wrong section, please move it in another PR.

Reply:
Good catch, I sneaked it in as part of #3720

Reply:
:+1: